tests/test_score.py

```diff
@@ -1,5 +1,10 @@
-from timefold.solver.score import SimpleScore, HardSoftScore, HardMediumSoftScore, BendableScore
```

Comment:
TBH, the vast majority of Python Score type method names would be different than the Java method name

- `add`, `subtract`, `multiply`, `divide` will be `__add__`, `__sub__`, ...
   so this will work in Python:
```python
a = 2 * SimpleScore(5) + SimpleScore(2) # SimpleScore(12)
```

- `hardScore`, `softScore`, etc. are mapped to attributes